.github/workflows/secrel4prs.yml

```diff
@@ -1,4 +1,4 @@
-name: "SecRel check for PRs"
+name: "SecRel 4 pipeline"
```

Comment:
Before merging, I'll rename this file to secrel4.yml. I'm keeping the current name so I can run this workflow manually for testing purposes. 
Ticket #1005 will create modular SecRel workflows for PRs.